java/src/main/java/com/plaid/quickstart/QuickstartApplication.java

```diff
@@ -35,6 +36,10 @@
   // persistent data store.
   public static String paymentId;
 
+  private PlaidApi plaidClient;
+  private ApiClient apiClient;
+  public String env;
```

Comment:
could this be more specific? env can refer to a lot of things, maybe this should be `plaidEnv`?

Reply:
changed to plaidEnv